test/performance/throughput/ThroughputSubscriber.hpp

```diff
@@ -85,6 +86,7 @@ class ThroughputSubscriber
     // Test synchronization
     std::mutex command_mutex_;
     std::mutex data_mutex_;
+    std::mutex lecture_available_mutex;
```

Comment:
I think we can use `data_mutex_` for this

Reply:
They are used for two different Listener functions and for different purpose, but as any of them can interlock each other I change it and leave just one.